README.md

```diff
@@ -1 +1,49 @@
+ * cryptographic data-structures, algorithms, languages and APIs required for the continued upkeep of the Polkadot (Main) Network;
+ * consensus algorithms concerning the Relay-chain (BABE \& GRANDPA);
+ * trust-free bridges relying on said consensus algorithms (planned to be) utilised by system chains;
+ * parachain consensus;
+ * cross-chain message passing (XCMP, HRMP, DMP \& UMP);
+ * the Polkadot libp2p-based peer networking protocol;
+ * the Polkadot topology strategies;
+ * chain synchronisation strategies utilised by Polkadot;
+ * the Polkadot business-logic (aka the 'runtime');
+ * pallets utilised by the Polkadot (Main) Network and its system chains;
+ * the internals of the frame pallet framework;
+ * runtime and host APIs;
+ * the XCM specification and realisation;
+ * standard RPCs;
+ * user-interface code required to practically execute upgrades to the Polkadot (Main) Network; and
+ * code or technology required by, and utilised primarily for, any code or technology already included.
+
+These RFCs are scoped to the subset of these concerns which must be held consistent across all implementations. Various implementation details, such as internal node algorithms, programming languages, or database formats are out of scope. Changes to network protocol descriptions, runtime logic and runtime public interfaces, inherents, transaction formats should be discussed via RFCs.
+
+## Significance 
+
+These RFCs are in practice only a signaling mechanism to determine and indicate the Fellowship's design and architecture preferences and to coordinate discussion and social consensus on architectures and designs according to open-source principles.
+
+The Fellowship holds only the powers vested in it by Polkadot's governance, which are limited to the expression of expert opinion and the ability to move proposals to more lenient governance tracks when necessary. It is not an arbiter of the "correctness" of any particular runtime or node implementation, and the practical meaning of these RFCs follows as a consequence of its limited powers. 
+
+For any RFC concerning runtime logic or interfaces, the Fellowship's capabilities are bounded by relay-chain governance, which is the ultimate decider of what code is adopted for block processing. As such, these RFCs are only loosely binding - the chains' governance has no obligation to accept the features as implemented and may accept features which have not gone through the RFC process. When it comes to node-side areas of expertise, the Fellowship's vote is more strongly binding, as the governance systems of the chains can't determine the environment the runtime is executed within, and in practice all node implementations should conform to some foundational standards in order to communicate.
+
+Merged RFCs are only an indication of support for a specific design, not a commitment to an implementation of a feature on any particular timeframe or roadmap ordering.
```

Comment:
So we will then need some "external process" to come to agreement when certain protocol changes will be implemented?